src/components/BrewList.js

```diff
@@ -1,21 +1,30 @@
 import React, { useState } from 'react'
 import './css/BrewList.css'
 import Card from './Card'
+import BrewDetails from '../BrewDetails'
+import { Routes, Route, Link } from 'react-router-dom'
+
 
 const BrewList = (breweries) => {
+  const [list, setList] = useState([])
 
   // console.log(breweries, 'breweries')
 
   const brewyDetails = breweries.breweries.map(brewy => {
-
+     
     return (
-      <Card
-          key={brewy.id}
-          type={brewy.brewery_type}
-          name={brewy.name}
-          city={brewy.city}
-          street={brewy.street}
-      />
+      <>
+        <Link to={`/${brewy.id}`} key={brewy.id} className='link-card'>
```

Comment:
I like that you have the Link right on the card like this! Instead of one layer deeper